mkemlog.c

```diff
@@ -33,11 +32,20 @@ int main(int argc, char** argv) {
     char* file;
     char* number;
     char* end_of_number;
+    int emlog_max_size;
     uid_t uid = -1;
     if (argc < 2 || argc > 5) {
         error(1 ,0, USAGE);
     }
     file = argv[1];
+
+    FILE *max_size_file = NULL;
+    max_size_file = fopen("/sys/module/emlog/parameters/emlog_max_size", "r");
+    if (errno)
+        error(1, errno, "Emlog module not loaded\n");
+    fscanf(max_size_file, "%d", &emlog_max_size);
+    if (errno)
+        error(1, errno, "Unable to get emlog max size\n");
     if (argc > 2 ) {
```

Comment:
Not really necessary, but would by nice to tidy up and do `fclose(max_size_file)` after we're done with it.